# HYPOTHESIS TESTING

##  Problem Statement

Bombay Hospitality Ltd. operates a franchise model for producing exotic Norwegian dinners. The weekly operating cost is given by:

=> W = 1000 + 5X

where \( X \) is the number of units produced in a week.

Recently, franchise owners claim that the actual weekly operating costs have increased beyond this model.
We’ll use hypothesis testing to investigate this claim.


##  Given Data

- Theoretical cost model: ( W = 1000 + 5X )
- ( X = 600 ) units  
- ( mu = 1000 + 5 times 600 = 4000 )
- Sample of 25 restaurants:
  - Sample Mean (( bar{x} )) = ₹3050
  - Standard Deviation of cost per unit produced: ₹5 × 25 = ₹125
  - Sample Size (( n )) = 25
- Significance level: (alpha = 0.05)


## Hypothesis Statement

###  Null Hypothesis (H₀):
The average weekly operating cost is ₹4000  
### Alternative Hypothesis (H₁):
The average weekly operating cost is higher than ₹4000  

#### So the hypotheses are:

H₀: μ = 4000

H₁: μ > 4000


## Test Type  

- One-tailed Z-test (as population standard deviation is known)
- Since H₁ is checking greater than ( > ), it’s a right-tailed test.

In [11]:
### Import Libraries
import math 
from scipy.stats import norm

In [12]:
# Define Data and Hypotheses
sample_mean = 3050        # x̄
theoretical_mean = 4000   # μ (from cost model)
std_dev = 125             # σ
n = 25                    # sample size
alpha = 0.05              # significance level

In [20]:
# Calculate Test Statistic
test_statistic = (sample_mean - theoretical_mean) / (std_dev / math.sqrt(n))
test_statistic

-38.0

In [21]:
# Determine Critical Value
critical_value = norm.ppf(1-alpha)
critical_value

1.6448536269514722

In [28]:
# Final Conclusion
if test_statistic > critical_value:
    decision = "Reject Null Hypothesis, There is strong evidence to support the claim that weekly operating costs are higher than the model suggests."
else:
    decision = "Failed to Reject Null Hypothesis, There is insufficient evidence to support the claim that weekly operating costs are higher than the model suggests."

print(decision)

Failed to Reject Null Hypothesis, There is insufficient evidence to support the claim that weekly operating costs are higher than the model suggests.


## Conclusion:
Based on the hypothesis test conducted:

The calculated test statistic (Z) is -38.00

The critical value for a right-tailed test at α = 0.05 is 1.645

Since -38.00 < 1.645, we fail to reject the null hypothesis

Therefore, there is no strong evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests.

In fact, the sample mean of ₹3050 is much lower than the expected ₹4000 — suggesting that costs may have decreased, not increased.